packages/prosemirror-suggest/src/suggest-utils.ts

```diff
@@ -162,6 +170,7 @@ function findMatch<Schema extends EditorSchema = EditorSchema>(
     supportedCharacters,
     multiline,
     caseInsensitive,
+    unicode,
```

Comment:
Looks like you need to add `unicode` to the `Pick` [here](https://github.com/remirror/remirror/blob/4bff1d2a7128372b87970787b7e6adc4b8615837/packages/prosemirror-suggest/src/suggest-utils.ts#L616)

Reply:
Yep, fixed - thanks a lot!